CHANGELOG.rst

```diff
@@ -3,6 +3,9 @@
 
 **New Features**
 
+* Support nose-style ``__test__`` attribute on methods of classes.
```

Comment:
I think it is worth mentioning that this is now working also for `TestCase` subclasses. For simple test classes setting `__test__` already works.